SafeExecute/hook.cpp

```diff
@@ -707,7 +707,7 @@ bool WINAPI CreateTimerQueueTimer_Hook(
     PreHook(1, "CreateTimerQueueTimer");
     
     // CreateTimerQueueTimer
-    res = MsgBox("Timer creation detected\nContinue execution?");
+    res = MsgBox("Timer creation detected\nThis function is often used in malware to avoid being analyzed.\nIn this case, malware calls a callback function every time specified (e.g., 1 second) by a timer, and uses that callback function to check whether it is being analyzed or not.\nContinue execution?");
```

Comment:
ああ～ でもこれ確かに説明むずいよね...
callbackを使うのはタイマー自体そういうものだから本質ではないんだけど、例えばこんな感じかな
`Timer creation detected\nTimer is often used in malware to avoid being detected in analysis environment.\n.In particular, this delays malicious execution until auto-analysis program finishes it's detection.\nContinue execution?`

Reply:
あ～そういうことなんですね！!
修正します！

Reply:
修正しました～

Reply:
何度も修正してもらって申し訳ない、
あざます！